plenum/server/pool_req_handler.py

```diff
@@ -180,44 +180,30 @@ def isNodeDataSame(self, nodeNym, newData, isCommitted=True):
         nodeInfo.pop(f.IDENTIFIER.nm, None)
         return nodeInfo == newData
 
-    def isNodeDataConflicting(self, data, updatingNym=None):
+    def isNodeDataConflicting(self, new_data, updating_nym=None):
         # Check if node's ALIAS or IPs or ports conflicts with other nodes,
         # also, the node is not allowed to change its alias.
 
         # Check ALIAS change
-        nodeData = {}
-        if updatingNym:
-            nodeData = self.getNodeData(updatingNym, isCommitted=False)
-            if nodeData.get(ALIAS) != data.get(ALIAS):
+        if updating_nym:
+            old_alias = self.getNodeData(updating_nym, isCommitted=False).get(ALIAS)
+            new_alias = new_data.get(ALIAS)
+            if old_alias != new_alias:
                 return True
```

Comment:
We just return True here, so for the client it's not clear what exactly is wrong (if this is non-unique IP, port or alias)